src/System.ComponentModel.Annotations/tests/MaxLengthAttributeTests.cs

```diff
@@ -58,6 +66,34 @@ public static void Ctor_Int(int length)
         }
 
         [Theory]
+        [MemberData(nameof(ValidValues_ICollection))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "MaxLengthAttribute in the .NET Framework doesn't support ICollection.Count. See https://github.com/dotnet/corefx/pull/2650")]
```

Comment:
Likewise for these behavior differences. (#2650 doesn't seem to have an issue associated, unless I missed it.)